src/lib/y2storage/filesystems/type.rb

```diff
@@ -152,6 +154,17 @@ def self.legacy_home_filesystems
         LEGACY_HOME_FILESYSTEMS.map { |f| find(f) }
       end
 
+      # Allowed filesystems for zipl boot partition
+      #
+      # @note See page 13 in following link
+      # https://share.confex.com/share/123/webprogram/Handout/\
+      #   Session15694/SHARE_Bootloader_Ihno_PittsPPT_0.09.pdf
+      #
+      # @return [Array<Filesystems::Type>]
+      def self.zipl_filesystems
```

Comment:
It should be mentioned that the first one is the preferred one (i.e. used by the guided setup)